### ISSUE TITLE
Unhandled Character Preview atom (examining multiple people fills up dreamseeker's object limit intensely)

### ISSUE THREAD

**Issue by DragonTrance** [Skyrat-SS13/Skyrat-TG#23451]
Byond has an object limit, a maximum amount of atoms that can be rendered at a given moment. **_The character preview while examining people adds to this atom count and starts to delete the UI after too long,_** as well as the tiles above you. This character preview when examining people is not deleted, even after closing the window and reconnecting, resulting in your object limit being used up for every atom that has to be rendered.

(This prevented me from using internals at all with UI, but I'm not sure if there's a button somewhere else to turn them on)

Round ID: 9299

Testmerges
#22097
#23050
#22799

## Reproduction:
You can either do:
1: Open Character Preferences, and reconnect while the window is open. Doing this repetitively will quickly hide the tiles above you.
or 2: In a live server, open the examine window for a lot of people. The tiles above you will start to disappear after a while when examining new people

Repeat this process, and eventually either the UI will disappear, or the tiles on the floor will start to disappear, from right-to-left, up-to-down

Eventually, if you notice tiles disappear, you can open the character preferences window and close it (WITHOUT disconnecting) to show this is handled correctly in a realistic scenario.

However, the issue I'm showing, is that _**examine previews are not handled the same way**_ and eventually add to the object limit in dreamseeker.

## Visual Examples
These are screenshots while inside the main server

Image 1:
Note how the tiles on the top of the screen are invisible. The actions UI is also gone completely
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/26515331/62402be8-1087-4097-9215-63a343acea6a)

Image 2:
After opening game preferences the same way, more tiles disappear
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/26515331/1e49ee88-2017-48dc-abe7-8629895e9311)

Video example:
Instances of examining people you already examined do not add to this limit, very respectfully, however examining new people displays the same problem, a tile is eaten up and isn't un-eatened when the window closes, showing that dreamseeker's object limit is still being used
(note the tiles on the top right getting used after examining one of the people in the shuttle brig, making me unable to examine the person on the opposite side of them after the tile gets used up)

https://github.com/Skyrat-SS13/Skyrat-tg/assets/26515331/76f76b48-c062-4c07-86bd-793de54959b2

**Comment by DragonTrance** [comment]
haha oops accidentally submitted while round was open

**Comment by DragonTrance** [comment]
Not sure if it's also related, but my performance continuously became more and more shit as time went on

**Comment by GoldenAlpharex** [comment]
Once you hit that, if you'd close all of those UIs and then reconnect, would it fix it?

**Comment by DragonTrance** [comment]
No, it doesn't fix it. If it did, I wouldn't be making this issue
